src/Watcher.php

```diff
@@ -34,7 +34,7 @@ public function __construct(array $folders)
      */
     public function getAllWatchedFiles() : array
     {
-        return array_values(array_flip($this->fileCache));
```

Comment:
The `$this->fileCache` will be a associate array with different key to assign the same values.

The sample associate array is as follows:

```
array(3) {
  '/data/watch-phpunit/tests/Fixtures/watchable/nested/nested-deeper/three.php' =>
  int(1564337690)
  '/data/watch-phpunit/tests/Fixtures/watchable/two.php' =>
  int(1564337690)
  '/data/watch-phpunit/tests/Fixtures/watchable/one.php' =>
  int(1564337690)
}
```

- The `array_flip` will exchange keys and values.
- It will output a associate array that is one key with one value because the values are same.

To avoid this behavior, using the `array_keys` instead.

This function can return all keys and return the result directly, then it will be expected behavior is `3`.